tuxemon/states/choice/choice_item.py

```diff
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: GPL-3.0
+# Copyright (c) 2014-2023 William Edwards <shadowapex@gmail.com>, Benjamin Bean <superman2k5@gmail.com>
+from __future__ import annotations
+
+from collections.abc import Callable, Sequence
+from typing import Any
+
+import pygame_menu
+from pygame_menu.locals import POSITION_CENTER, POSITION_EAST
+from pygame_menu.widgets.selection.highlight import HighlightSelection
+
+from tuxemon import prepare
+from tuxemon.db import db
+from tuxemon.menu.menu import PygameMenuState
+from tuxemon.menu.theme import get_theme
+from tuxemon.tools import transform_resource_filename
+
+ChoiceMenuGameObj = Callable[[], None]
+
+
+def fix_width(screen_x: int, pos_x: float) -> int:
+    """it returns the correct width based on percentage"""
+    value = round(screen_x * pos_x)
+    return value
```

Comment:
The statements can be inlined, i.e. we don't need to store the value in a temporary variable that's going to be returned in the line anyway.
```suggestion
    return round(screen_x * pos_x)
```

Reply:
Yeah, see reply above